mono/mini/interp/transform.c

```diff
@@ -4160,6 +4165,13 @@ mono_interp_transform_method (RuntimeMethod *runtime_method, ThreadContext *cont
 		offset += size;
 	}
 	offset = (offset + 7) & ~7;
+
+	runtime_method->exvar_offsets = g_malloc (header->num_clauses * sizeof (guint32));
+	for (i = 0; i < header->num_clauses; i++) {
+		offset += sizeof (MonoObject*);
+		runtime_method->exvar_offsets [i] = offset;
+	}
```

Comment:
ok.  I just noticed we align `offset` above, I think we should do that after computing the slots for `exvar_offsets`.